ote/src/cljc/ote/db/common.cljc

```diff
@@ -10,5 +10,12 @@
      (:require-macros [ote.db.specql-db :refer [define-tables]])))
 
 (define-tables
+  ;; Enums
+  ["stat_type" ::stat_type (specql.transform/transform (specql.transform/to-keyword))]
+
   ;; Common UDT types
-  ["address" ::address])
+  ["address" ::address]
+
+  ;; Tables
+  ["stats-service" ::stats-service]
```

Comment:
If stats-service is a generic table, shouldn't it and its types be defined in a new `ote.db.stats`?
(or if it's netex-specific it could go into `ote.db.netex`)

Reply:
Maybe we should define new ote.db.stats because it will be used in many cases. It is more related to service (stats-service) than netex and that is why I would not put it in ote.db.netex.

I thought to add it to ote.db.transport-service but I think that there is just too much data already. 

If ote.db.common is the wrong place I can move it.

Reply:
Agreed, only pro's when putting in ote.db.stats